test/client_test.js

```diff
@@ -18,14 +18,32 @@ describe('Client', () => {
     expect(this.fhirClient.baseUrl).to.deep.equal(new URL(baseUrl));
   });
 
+  describe('invalid instance', () => {
+    const baseUrl = 'http://test.com';
+
+    it('responds to #authMetadata(), but returns false due to missing SMART OAuth URIs', async function () {
+      nock(baseUrl)
+        .matchHeader('accept', 'application/json+fhir')
+        .get('/metadata')
+        .reply(200, () => fs.createReadStream(path.normalize(`${__dirname}/fixtures/no-smart-oauth-uri-capability-statement.json`, 'utf8')));
+
+      const config = { baseUrl };
+      this.fhirClient = new Client(config);
```

Comment:
Since setting up `fhirClient` is in the same `it` assertion block (i.e. and not outside scope, such as a `beforeEach`/`beforeAll` block), there is no need to assign to the `this` mocha object.

In other words, I think we should use local vars here:

```javascript
const fhirClient = new Client(config);
const authMetadata = await fhirClient.smartAuthMetadata();
...
```